admin/src/Games/GameList.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+	ChipField,
+	Datagrid,
+	DateField,
+	List,
+	ReferenceField,
+	ReferenceManyField,
+	SingleFieldList,
+	TextField,
+} from "react-admin";
+import { GameStatusField } from "./GameStatusField.tsx";
+
+export function GameList() {
+	return (
+		<List
+			resource="games"
+			sort={{
+				field: "updated_at",
+				order: "DESC",
+			}}
+			queryOptions={{
+				meta: {
+					columns: ["*", "status", "winner_name"],
+				},
+			}}
+		>
+			<Datagrid>
```

Comment:
Same, can you use a `DataTable` instead? Besides, this is why you currently don't have a label for the status column